src/utils/cell.ts

```
@@ -17,13 +28,41 @@ function renderCell(
-  const cell = new Graphics(context);
+  const activeContext = base.clone().fill(colors.active);
+
+  const cell = new Graphics(inactiveContext);
+
+  cell.eventMode = "static";
+
+  cell.hitArea = new Rectangle(
+    horizontalShift,
+    verticalShift,
+    squareArea,
+    squareArea
```

ESLint

[error] Insert `,`

(prettier/prettier)